src/Memory.php

```diff
@@ -22,10 +22,11 @@ class Memory implements Memoize
      * @param string   $key
      * @param callable $compute
      * @param int|null $cacheTime
+     * @param bool     $shouldUpdate
      *
      * @return mixed
      */
-    public function memoizeCallable(string $key, callable $compute, int $cacheTime = null)
+    public function memoizeCallable(string $key, callable $compute, int $cacheTime = null, $shouldUpdate = false)
     {
         if (array_key_exists($key, $this->cache)) {
```

Comment:
Should this also be updated?

```suggestion
    public function memoizeCallable(string $key, callable $compute, int $cacheTime = null, $refresh = false)
    {
        if (array_key_exists($key, $this->cache) && !$refresh) {
```

Reply:
updated